MerlinAU.sh

```diff
@@ -4097,6 +4100,27 @@ _EntwareServicesHandler_()
    then return 0 ; fi  ## Entware is NOT found ##
 
    serviceStr="$(/usr/bin/find -L "$entwOPT_init" -name "S*" -exec ls -1 {} \; 2>/dev/null | /bin/grep -E "${entwOPT_init}/S[0-9]+")"
+
+   # Filter out services to skip and add a skip message #
+   for skipService in $skipServiceList
+   do
+       skippedService="$(echo "$serviceStr" | /bin/grep -E "S[0-9]+.*${skipService}$")"
+       if [ -n "$skippedService" ]
+       then
+           skippedServiceFile="$(basename "$skippedService")"
+           Say "Skipping $skippedServiceFile $1 call..."
+
+           # Rename service file so it's skipped by Entware #
+           if mv -f "${entwOPT_init}/$skippedServiceFile" "${entwOPT_init}/OFF.${skippedServiceFile}.OFF"
+           then
+               [ -z "$skippedServiceList" ] && \
+               skippedServiceList="$skippedServiceFile" || \
+               skippedServiceList="$skippedServiceList $skippedServiceFile"
+               serviceStr="$(echo "$serviceStr" | /bin/grep -vE "S[0-9]+.*${skipService}$")"
+           fi
+       fi
+   done
+
    [ -n "$serviceStr" ] && serviceCnt="$(echo "$serviceStr" | wc -l)"
```

Comment:
I believe this to be the issue here.

He only has the tailscale service, which means this counts the remaining services to be stopped. If no services are found, it exits. But it shouldn't because in his case it skipped tailscale, found no other services, exited, and didn't complete the rename back even though it renamed it to "OFF" already.

Reply:
Next when he tries to do: _EntwareServicesHandler_ start, it can't find the service because it's called *OFF*, even if it can, starting it at this point would break it since it was never stopped in the first place.

Reply:
> I believe this to be the issue here.
> 
> He only has the tailscale service, which means this counts the remaining services to be stopped (after already skipping/renaming the tailscale service).
> 
> If no services are found, it exits.
> 
> But it shouldn't I don't think, because in his case it skipped tailscale, found no other services, exited, which means it didn't complete the rename back, even though it renamed it to "OFF".
>
Very good catch!! That's why I always advocate for having a 2nd pair of eyes to do testing & validation.

Reply:
You are my second set of eyes, and I am yours!